omnigan/data.py

```diff
@@ -167,6 +179,7 @@ def check_samples(self):
         exist on the file-system
         """
         for s in self.samples_paths:
+            assert all([t in s for t in self.tasks])
```

Comment:
actually I'm not sure we should have that here 'cause it prevents samples from having missing data. what do you think?  should we enforce all samples having the same data, matching `opts.tasks` ?